spec/game_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../lib/game.rb'
+
+RSpec.describe Game do
+  let(:array_1) { %w[X X X _ _ _ _ _ _] }
+  let(:array_2) { %w[_ _ _ X X X _ _ _] }
+  let(:array_3) { %w[_ _ _ _ _ _ X X X] }
+  let(:array_4) { %w[X O X _ _ _ _ _ _] }
+  let(:array_5) { %w[X _ _ X _ _ X _ _] }
+  let(:array_6) { %w[_ O _ _ O _ _ O _] }
+  let(:array_7) { %w[_ _ O _ _ O _ _ O] }
+  let(:array_8) { %w[O _ _ _ O _ _ _ O] }
+  let(:array_8) { %w[_ _ O _ O _ O _ _] }
+  let(:array_9) { %w[_ _ X O _ O _ _ _] }
+  let(:array_10) { %w[_ _ _ O _ O _ _ O] }
+  let(:game) { Game.new }
+
+  describe '#winning_condition' do
+    context 'when the condition is winning' do
```

Comment:
A condition can not win. Eighter it can pass or fail. So instead of 'when the condition is winning', I would rather say 'When the list of inputs generate a winning combination'